1847 scylla manager 3.0

mrlexor opened the pull request:
`scylla_manager_task_*` metrics have been replaced on new ones: https://github.com/scylladb/scylla-monitoring/issues/1847

mrlexor committed:
scylla_manager_task metrics have been replaced

`scylla_manager_task_*` metrics have been replaced to new ones according on new metrics in new version of `scylla-manager`

mrlexor committed:
scylla_manager_task metrics have been replaced

`scylla_manager_task_*` metrics have been replaced according on new metrics in new version of `scylla-manager`

amnonh commented on grafana/scylla-manager.template.json:
```diff
@@ -142,13 +142,6 @@
-                              "format": "time_series",
-                              "intervalFactor": 2,
-                              "dashversion":"<2.4",
-                              "refId": "A"
```
You don't need to remove this part, if your change is for 3.0 this section is only valid for 2.3 or lower, that's what the dashversion is doing.

mrlexor replied:
I've returned back the expressions. I deleted them because they were existed in my dashboard which I built for version 3.0, but I did it directly execute python script, so, probably, somewhere I had mistake and the expressions appeared in the dashboard after building.

amnonh commented on grafana/scylla-manager.template.json:
```diff
@@ -256,13 +249,6 @@
-                              "expr": "sum(avg(scylla_manager_backup_percent_progress{cluster=~\"[[cluster]]\", instance=\"\", keyspace=\"\"}) by (task) * sum(scylla_manager_task_active_count{cluster=~\"[[cluster]]\", type=\"backup\"}) by (task)) or vector(0)",
-                              "format": "time_series",
-                              "intervalFactor": 2,
-                              "dashversion":"<2.4",
```
Same here

amnonh commented:
hi @mrlexor congrats on the PR.
beside my commetns, make sure you move it out of draft so it can be merged.
Also, add a **Fixes** to the comment with the issue the PR fixes

mrlexor committed:
Update scylla-manager.template.json

Expressions related to `backup` and `repair` tasks have been returned back due to defined `dashversion` that compatible with `scylla-manager` 2.4 and lower.

amnonh commented:
Fixes #1847
